cmd/index/capabilities/command.go

```diff
@@ -71,6 +71,8 @@ func NewCmd() *cobra.Command {
 		"Name of Kubernetes Secret to use for pulling registry images")
 	cmd.Flags().StringVar(&flags.ServiceAccount, "service-account", "default",
 		"Name of Kubernetes Service Account to use")
+	cmd.Flags().StringVar(&flags.NameSpace, "namespace", "default",
+		"Name of Kubernetes Namespace to use")
```

Comment:
I see that the namespace flag doesn't have a short name. I would consider using StringVarP for posix format with possibly -n as the short name for namespace.